doc/source/installation.rst

```diff
@@ -121,9 +127,27 @@ Please see :ref:`docker-usage-example` for usage.
 Source
 ======
 
-Due to the rapid pace of development on this tool, you might want to install it
-in "`development mode`_" so that new updates can be obtained by simply doing a
-``git pull`` in the repository's directory.
+Due to the rapid pace of development on this tool, you might want to
+install and update a bleeding-edge version of Molecule from Git.
+
+Follow the instructions below to do the initial install and subsequent
+updates.
+
+The package distribution that you'll get installed will be autogenerated
+and will contain a commit hash information making it easier to refer to
+certain unstable version should the need to send a bug report arise.
+
+.. warning::
+
+  Please avoid using ``--editable``/``-e`` `development mode`_ when
+  installing Molecule with Pip. This not very well supported and only
+  needed when doing development.
+  For contributing purposes, it's already integrated with ``tox``
+  properly.
+
+  .. _`development mode`:
+     https://setuptools.readthedocs.io/en/latest\
```

Comment:
Can be an intersphinx link instead please.

Reply:
It'd be an unrelated change. This was in the original doc version. I'd like to keep only related changes in this PR.